src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/repository/OffenderIndividualScheduleRepository.kt

```diff
@@ -49,6 +49,32 @@ interface OffenderIndividualScheduleRepository :
     last: Long,
   ): List<Long>
 
+  @Query(
+    """SELECT prisonId, eventSubType, pastOrFuture, COUNT(*) as appointmentCount FROM
```

Comment:
Do you not need the index hint? I think you will as the where clause is the same .